src/controllers/shared.controllers/director.name.controller.ts

```diff
@@ -11,22 +11,27 @@ import { CompanyAppointment } from "private-api-sdk-node/dist/services/company-a
 import { getCompanyAppointmentFullRecord } from "../../services/company.appointments.service";
 import { BASIC_STOP_PAGE_PATH, URL_QUERY_PARAM, DIRECTOR_NAME_PATH, UPDATE_DIRECTOR_NAME_PATH } from "../../types/page.urls";
 import { STOP_TYPE } from "../../utils/constants";
+import { formatTitleCase, retrieveDirectorNameFromAppointment } from "../../utils/format";
 
 export const getDirectorName = async (req: Request, res: Response, next: NextFunction, templateName: string, backUrlPath: string, isUpdate?: boolean) => {
   try {
     const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
     const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
     const session: Session = req.session as Session;
+    const officerFiling = await getOfficerFiling(session, transactionId, submissionId);
     const lang = selectLang(req.query.lang);
     const locales = getLocalesService();
     let currentUrl;
+    let directorName;
```

Comment:
Should this be retrieved from the filing for AP01? Here it's effectively always undefined for AP01 I think.

Reply:
This isn't retrieved/displayed on the name page for the AP01 journey, as the name hasn't been added to the filing yet and it isn't on the prototype. Is this something we need to add in case the user goes ahead in the journey and then comes back to this page?